storage/scripts/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS customers (
+    id VARCHAR(26) PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    email VARCHAR(50) NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE UNIQUE INDEX customer_email on customers(email);
```

Comment:
wouldn't it be better to pass the "UNIQUE NOT NULL" value directly into the email field in the table?